readme_assets/system_diagram/system_architecture.xml

```diff
@@ -0,0 +1,133 @@
+          <mxGeometry relative="1" as="geometry">
+            <mxPoint x="381.7391357421875" y="610" as="targetPoint" />
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-15" value="Load Balancer" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#dae8fc;strokeColor=#6c8ebf;" vertex="1" parent="1">
+          <mxGeometry x="525" y="590" width="120" height="40" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-16" value="frontend" style="rounded=0;whiteSpace=wrap;html=1;strokeWidth=1;fillColor=#f8cecc;strokeColor=#b85450;" vertex="1" parent="1">
+          <mxGeometry x="525" y="700" width="120" height="40" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-18" style="edgeStyle=orthogonalEdgeStyle;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;startArrow=classic;startFill=1;exitX=0.5;exitY=0;exitDx=0;exitDy=0;entryX=0.5;entryY=1;entryDx=0;entryDy=0;entryPerimeter=0;" edge="1" parent="1" source="JVIGxF9oKAkimninJbSp-16" target="JVIGxF9oKAkimninJbSp-15">
+          <mxGeometry relative="1" as="geometry">
+            <mxPoint x="584.5" y="700" as="sourcePoint" />
+            <mxPoint x="585" y="660" as="targetPoint" />
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-26" style="edgeStyle=orthogonalEdgeStyle;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;entryX=0.5;entryY=0;entryDx=0;entryDy=0;startArrow=classic;startFill=1;endArrow=classic;endFill=1;" edge="1" parent="1" source="JVIGxF9oKAkimninJbSp-21" target="JVIGxF9oKAkimninJbSp-10">
+          <mxGeometry relative="1" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-21" value="Redis" style="strokeWidth=1;html=1;shape=mxgraph.flowchart.database;whiteSpace=wrap;fillColor=#dae8fc;strokeColor=#6c8ebf;" vertex="1" parent="1">
+          <mxGeometry x="520" y="250" width="130" height="70" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-58" value="-Search queries&lt;br&gt;-Tag Stats&amp;nbsp; &amp;nbsp; &amp;nbsp; &amp;nbsp;&amp;nbsp;" style="text;html=1;align=left;verticalAlign=middle;resizable=0;points=[];autosize=1;strokeColor=none;fillColor=none;" vertex="1" parent="1">
+          <mxGeometry x="770" y="350" width="110" height="40" as="geometry" />
+        </mxCell>
+        <mxCell id="JVIGxF9oKAkimninJbSp-59" value="-Site cache&amp;nbsp; &amp;nbsp; &amp;nbsp; &amp;nbsp;&amp;nbsp;&lt;br&gt;-Traffic stats&amp;nbsp; &amp;nbsp; &amp;nbsp;&amp;nbsp;&lt;br&gt;-Dead link cache" style="text;html=1;align=left;verticalAlign=middle;resizable=0;points=[];autosize=1;strokeColor=none;fillColor=none;" vertex="1" parent="1">
```

Comment:
We no longer hold traffic stats anywhere as far as I am aware, so I think we can remove that bit.

```suggestion
        <mxCell id="JVIGxF9oKAkimninJbSp-59" value="-Site cache&amp;nbsp; &amp;nbsp; &amp;nbsp; &amp;nbsp;&amp;nbsp;&lt;br&gt;-Dead link cache" style="text;html=1;align=left;verticalAlign=middle;resizable=0;points=[];autosize=1;strokeColor=none;fillColor=none;" vertex="1" parent="1">
```